### PR TITLE
add loader to TagsQuery component

### PR DESCRIPTION
**Pull request by vstlouis**
This adds a spinner to the TagsQuery component. The loader shows when the node is updating.
Storybook was also added to show a loading TabQuery. Closes https://github.com/smartprocure/spark/issues/10086

Loading Tabs
Screen Shot 2021-04-20 at 8 42 47 PM

Storybook Loading Tags (Note that this is either pulled as a prop, from the theme, or the default Loader (Which this is currently doing which is why it looks different)
Screen Shot 2021-04-20 at 9 05 13 PM

### CODE REVIEW

**Comment by vstlouis** [comment]
I think we should disable the popover if it is loading (not being able to trigger it via a click if the node is updating), thoughts @stellarhoof @daedalus28

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>The version was not updated. Please update the version.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>The changelog has not been updated. Please update the changelog.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Branch being merged does not follow [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/)
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against da449a3b55c951af05b2124b443560939f7b45b5
</p>

**Comment by stellarhoof** [comment]
That blue on green looks ugghh

File: src/exampleTypes/TagsQuery/index.js [hunk 1]
```diff
@@ -19,17 +19,45 @@ let TagsQuery = ({
...
     let tagProps = {
       ...props,
       ...(!_.isNil(result)
         ? { label: `${props.value} (${toNumber(result)})` }
         : {}),
+      ...(node.updating
+        ? {
+            label: (
+              <>
+                {props.value}(
```
Comment (daedalus28) [inline src/exampleTypes/TagsQuery/index.js:39]: we should add a space before the parens like the above

File: src/exampleTypes/TagsQuery/index.js [hunk 2]
```diff
@@ -19,17 +19,45 @@ let TagsQuery = ({
   popoverPosition = 'bottom right',
   popoverArrow,
   popoverOffsetY,
-  theme: { Icon, TagsInput, Tag, Popover },
+  Loader,
+  theme: { Icon, TagsInput, Tag, Popover, Loader: ThemeLoader },
   joinOptions,
   ...props
 }) => {
+  Loader = Loader || ThemeLoader || StripedLoader
```
Comment (daedalus28) [inline src/exampleTypes/TagsQuery/index.js:27]: these shouldn't be necessary - the contexture react theme API allows you to override the theme components, so we should just use that

**Review by daedalus28 — CHANGES_REQUESTED** [review]
RE: @stellarhoof's comment - we already have the ability to customize it per component, so we can just make a generic color loader that gets injected for TagQuery specifically.

**Comment by daedalus28** [comment]
This is stale and no longer relevant

**Pull request closed without merging**